src/test/java/org/opentripplanner/api/resource/CalendarServiceDataStub.java

```diff
@@ -0,0 +1,36 @@
+package org.opentripplanner.api.resource;
+
+import org.opentripplanner.model.FeedScopedId;
+import org.opentripplanner.model.calendar.CalendarServiceData;
+import org.opentripplanner.model.calendar.ServiceDate;
+
+import java.util.Set;
+import java.util.TimeZone;
+
+/**
+ * This class extends the {@link CalendarServiceData} class to allow for easier testing.
+ * It includes methods to return both the set of service ids and the time zone used for testing.
+ */
+public class CalendarServiceDataStub extends CalendarServiceData {
+    private static final long serialVersionUID = 1L;
+
+    private final Set<FeedScopedId> serviceIds;
+    private final TimeZone timeZone;
+
+    public CalendarServiceDataStub(Set<FeedScopedId> serviceIds, TimeZone timeZone) {
+        this.serviceIds = serviceIds;
+        this.timeZone = timeZone;
+    }
+
+    @Override public Set<FeedScopedId> getServiceIds() {
```

Comment:
Not crazy about annotations on the same line.

Reply:
Me neither, I really ought to update my Intellij styling prefs.

Reply:
Changed in 7d5e6d4cf01b5e8212d232d4281329328d26d7f7